instadam/project.py

```diff
@@ -319,3 +324,39 @@ def update_user_permission(project_id):
         db.session.commit()
 
     return construct_msg('Permission added successfully'), 200
+
+
+@bp.route('/project/<project_id>', methods=['DELETE'])
+@jwt_required
+def delete_user(project_id):
```

Comment:
this function does not have a comment

Reply:
We will add comments to document this function